pkgs/by-name/ll/llama-cpp/package.nix

```diff
@@ -146,7 +156,7 @@ effectiveStdenv.mkDerivation (finalAttrs: {
     license = licenses.mit;
```

Comment:
The description also wants to be updated from the upstream: Inference of LLaMA model in pure C/C++

I like the `pnameSuffix` and `descriptionSuffix` things that upstream does as well, which I note you chose not to port over.

Reply:
I was a little lazy, and wasn't 100% sure it was going to be accepted. I wanted to make the smallest change possible. How about we do that in a follow up PR ?

Reply:
Yeah, I have no strong feelings about it being in this PR.